src/test/scala/com/gu/support/workers/errors/PayPalErrorsSpec.scala

```diff
@@ -10,10 +10,12 @@ import com.gu.support.workers.Fixtures.{createPayPalPaymentMethodDigitalPackJson
 import com.gu.support.workers.LambdaSpec
 import com.gu.support.workers.exceptions.RetryUnlimited
 import com.gu.support.workers.lambdas.CreatePaymentMethod
+import com.gu.test.tags.annotations.IntegrationTest
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
 
+@IntegrationTest
```

Comment:
Sorry if I'm being slow, but why does this need to be tagged as a result of this PR?

Reply:
I'm not sure it does actually. Don't know where that came from!